index.html

```diff
@@ -16,6 +16,11 @@
     <meta property="og:image:height" content="240" />
     <meta property="og:type" content="website" />
     <link rel="stylesheet" type="text/css" href="/typography.css" />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdn.jsdelivr.net/npm/katex@0.16.2/dist/katex.min.css"
+      crossorigin="anonymous" />
```

Comment:
We should bundle this with our app and not rely on a 3rd party cdn. We don't do that for any other of our assets.

Reply:
Done with 5026dcebe30dc655b3621f0bf4083e619dc97547

Reply:
We should make pretier ignore it:

```
Checking formatting...
[warn] public/katex.min.css
[warn] Code style issues found in the above file. Forgot to run Prettier?
Error: Process completed with exit code 1.
```

Reply:
Yeah went with the non minified version. Maybe even better if we want to debug something katex related